docs: update Device Settings and Dev Tools documentation (#1287)
software-mansion/radon-ide: packages/docs/docs/features/device-settings.mdx

1. **Device Appearance** - allows to change the light/dark mode.
2. **Text Size** - a slider that allows to change the font size on the device.

> 3. **Press Home Button** - simulates a press of a home button.

Reviewer: ```suggestion
3. **Press Home Button** - simulates a press of the home button.
```

> 4. **Open App Switcher** - option for quickly switching between opened apps.

Reviewer: ```suggestion
4. **Open App Switcher** - allows to quickly switch between opened apps.
```
the previous options are stated as verbs

> 5. **Biometrics** - options to simulate biometric authorization on iOS. This option isn't available on Android. The following options are available:

Reviewer: ```suggestion
5. **Biometrics** - options for simulating biometric authorization on iOS. This option isn't available on Android. The following options are available:
```

   <img width="250" src="/img/docs/ide_biometrics.png" className="shadow-image" />

>    - _Enrolled_ - whether the biometric authorization is enabled on the device,

Reviewer: ```suggestion
   - _Enrolled_ - controls whether the biometric authorization is enabled on the device,
```

</details>
<br/>

7. **Localization** - allows to set the device [localization](https://developer.mozilla.org/en-US/docs/Glossary/Localization). Changing the localization setting reboots the device. This option isn't available on Android.

<img width="450" src="/img/docs/ide_localization.png" className="shadow-image" />

8. **Reset Permissions** - allows to reset the device privacy permissions.

On iOS, you can reset all permissions, as well as reset location, photos, contacts and calendar. On Android, only resetting all permissions is supported.

<img width="250" src="/img/docs/ide_permissions.png" className="shadow-image" />

> 9. **Deep links** - allows to provide deep links and website URLs to the device. Website URLs will open in the default device browser.

Reviewer: ```suggestion
9. **Deep links** - allows opening deep links and website URLs on the device. Website URLs will open in the default device browser.
```

<img width="450" src="/img/docs/ide_deep_link.png" className="shadow-image" />

> 10. **Enable replays** - a toggle for enabling replays. After switching the `Enable Replays` toggle a `Replay` button appears in the Radon IDE panel.

Reviewer: ```suggestion
10. **Enable replays** - enables or disables the "replay" functionality. After switching the `Enable Replays` toggle a `Replay` button appears in the Radon IDE panel.
```
